src/codegen/compile.ml

```diff
@@ -4164,7 +4164,81 @@ module RTS_Exports = struct
 
 end (* RTS_Exports *)
 
-module Serialization = struct
+module type Stream = sig
+  (*               env    ref    code *)
+  val write_byte : E.t -> G.t -> G.t -> G.t
+  val write_word_leb : E.t -> G.t -> G.t -> G.t
+  val write_word_32 : E.t -> G.t -> G.t -> G.t
+  val write_blob : E.t -> G.t -> G.t -> G.t
+  val write_text : E.t -> G.t -> G.t -> G.t
+  val write_unsigned : E.t -> G.t -> G.t -> G.t
```

Comment:
```suggestion
  val write_bignum_unsigned : E.t -> G.t -> G.t -> G.t
```
maybe?

Reply:
`write_bignum_leb` and `write_bignum_sleb`?

Reply:
Done: 5c4819fec